opensir/models/model.py

```diff
@@ -28,8 +27,8 @@ class Model:
     """ Base model definition """
 
     CSV_ROW = []
-    NUM_PARAMS = 4
-    NUM_IC = 4
+    PARAMS = []
```

Comment:
I like this. We avoid magic numbers :)

Reply:
I like avoiding magic numbers in the model mother class, but let's keep it in the sir and sirx models as the magic numbers are natural to the systems.